DeviceSecurityTestPkg/SpdmDeviceSecurityDxe/SpdmDeviceSecurityDxe.c

```diff
@@ -436,7 +436,7 @@ CreateSpdmDriverContext (
   SpdmSetData (SpdmContext, SpdmDataCapabilityCTExponent, &Parameter, &Data8, sizeof (Data8));
 
   Data32 = 0 |
-           //           SPDM_GET_CAPABILITIES_REQUEST_FLAGS_CERT_CAP |
+           SPDM_GET_CAPABILITIES_REQUEST_FLAGS_CERT_CAP |

```

Comment:
This is for mutual authentication. Why it is needed?

Reply:
please remove SPDM_GET_CAPABILITIES_REQUEST_FLAGS_KEY_EX_CAP, SPDM_GET_CAPABILITIES_REQUEST_FLAGS_PSK_CAP_REQUESTER, SPDM_GET_CAPABILITIES_REQUEST_FLAGS_HBEAT_CAP, SPDM_GET_CAPABILITIES_REQUEST_FLAGS_KEY_UPD_CAP, SPDM_GET_CAPABILITIES_REQUEST_FLAGS_HANDSHAKE_IN_THE_CLEAR_CAP